peps/pep-0772.rst

```diff
@@ -263,48 +267,47 @@ During a council term, if changing circumstances cause this rule to be broken
 council members must resign to remedy the issue, and the resulting vacancies
 can then be filled as normal.
 
--------------------
-Packaging community
--------------------
+--------------
+Voting members
+--------------
 
 Responsibility
 ==============
 
-Packaging community members participate in formal votes to elect the Packaging
-Council.
+Voting members participate in formal votes to elect the Packaging Council.
+
 
 Processes
 =========
 
 Initial membership
 ------------------
 
-Initial membership in the Packaging community will include anyone who has taken
-the time to formalise their participation in the Packaging community. This
-includes:
+Initial set of voting members will include anyone who has taken the time to
+formalise their participation in the packaging community. This includes:
 
 * PyPA members: Anyone with the triage bit or commit bit, or at least one
   project in the PyPA organisation.
 * Packaging workgroup members: Anyone who is listed on the Packaging WG charter
-  will be moved into the Packaging community.
+  who is willing to participate is welcome.
 * Interested core team members: Any Python core team member who is willing to
   participate is welcome.
-* Wider community members: Non-profit organisations that participate in
-  packaging or working with new packagers. For example, PyOpenSci, NumFocus,
-  Django, are encouraged to initially nominate up to seven members by sending
-  an email to \[todo\].
+* Wider community members: An initial set of for-profit companies, nonprofit
+  organizations, academic or educational institutions and smaller unaffiliated
+  projects would be invited to nominate three individuals to represent them.
+  [TODO: add a link here for the initial set, that the authors curate]
```

Comment:
This is intentionally left as a TODO.

Reply:
For history/archival reasons, can we have the eventual full list in the PEP itself, rather than an external resource?

Reply:
I'm keeping notes and will add this section to a follow up PR.  For now, I'm going to resolve this thread and get this PR merged.